crack.gemspec

```diff
@@ -14,4 +14,6 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = Crack::VERSION
   gem.license       = "MIT"
+
+  gem.add_runtime_dependency("rexml")
```

Comment:
What affect does this have on older versions of Ruby? Looks like rexml has been a gem for two years at least https://rubygems.org/gems/rexml/versions